packages/config/tailwind/index.js

```diff
@@ -1,6 +1,12 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  content: ["./src/**/*.{ts,tsx}", "./src/_app.tsx"],
+  content: [
+    // NextJS
+    "./src/**/*.{ts,tsx}",
+    // Expo
+    "./app/*.{ts,tsx}",
+    "./app/**/*.{ts,tsx}",
+  ],
```

Comment:
Just include `src/**` here then expo Will have to extend the preset and handle "it's special case".

Reply:
Alright, will do it today

Reply:
Implemented :)